Connect to devnet: NFT feature stanza

mDuo13 opened the pull request:
- Updates the required `[features]` stanza for connecting to Devnet to have the NFT feature that's expected there soon
- Removes $ from shell commands to facilitate copy-pasting

mDuo13 committed:
Connect to devnet: NFT feature stanza

manojsdoshi commented:
LGTM

manojsdoshi reviewed (approved):
LGTM

mDuo13 committed:
NFT stanza feature rename

tequdev commented:
https://github.com/XRPLF/xrpl-dev-portal/blob/361209e027c758f60e81535f0ae5e50b79b27b00/content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md?plain=1#L46

Can you please fix this ordered list ?
<br>
```
3. Comment out the previous `[ips]` stanza, if there is one:
```
to

```
1. Comment out the previous `[ips]` stanza, if there is one:
```

mDuo13 committed:
NFT-Devnet details

manojsdoshi reviewed (approved):
LGTM

mDuo13 commented:
Updated:

- NFTs will go to their own network ("NFT-Devnet") not the current Devnet. Added the relevant details in a lot of places.
- Fixed #1250.
- Fixed the numbering typo @develoQ pointed out.

- [ ] TODO: UNL address & keys for the NFT-Devnet. @manojsdoshi is working on that over the weekend.

mDuo13 commented on content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md:
```diff
@@ -83,8 +117,18 @@ Edit your `validators.txt` file. This file is located in the same folder as your
+        TODO
+
+        [validator_list_keys]
+        TODO
```
Note: Make these not "TODO" before merging

mDuo13 committed:
Final values for NFT-Devnet stuff
